assets/css/main.css

```diff
@@ -2,6 +2,10 @@ body {
   font-family: "Philosopher";
 }
 
+.sidebar .scrolling {
+  display: none;
```

Comment:
What's this line for?

Reply:
That I added while testing and forgot to remove afterwards. Can I submit another pull request after removing the same. 
Is there anything else that needs to be changed?

Reply:
you can just commit the changes to the same branch. No need to create a new PR.

Reply:
did the same. Please have a look.